bigbio/biodatasets/tmvar_v2/tmvar_v2.py

```diff
@@ -223,6 +223,14 @@ def parse_raw_doc(self, raw_doc):
                     entity_id,
                 ) = mentions
                 rsid = None
+                if pmid == 18166824 and count == 0:
+                    # Setting rsid to entity_id, entity_id to semantic_type_id and semantic_type_id to "ProteinMutation"
+                    entity_id_temp = semantic_type_id
+                    rsid = entity_id
+                    entity_id = entity_id_temp
+                    semantic_type_id = "ProteinMutation"
+                    logger.warning("Semantic type missing. ProteinMutation inserted")
+                    logger.warning(f"Document ID: {pmid} Line: {line}")
```

Comment:
for our future selves, can we include a comment that shows the full line `18166824    880    948    amino acid (proline) with a polar amino acid (serine) at position 29    p|SUB|P|29|S    RSID:2075789` and states the semantic_type_id is missing and would typically be between `position 29` and `p|SUB|P|29|S` so future code readers know what's going on?

Reply:
The full line is already shown in the warning. I will add what's missing

Reply:
totally get it. but when I refer to "future selves" and "code readers" I mean people reading this code and wondering what's going on w/o running it ... for example, if someone besides me was reviewing this, they would have to load the dataset and see the line to know that the changes make sense ... and in the future someone else (or us) will probably be in that position and will not have looked at the output of the script for a long time.

Reply:
Oh yeah, completely agree. The newest commit should take care of it :)